usaspending_api/etl/management/commands/check_es_doc_duplication.py

```diff
@@ -0,0 +1,245 @@
+import logging
+import numpy as np
+
+from django.core.management.base import BaseCommand
+from elasticsearch import Elasticsearch
+from elasticsearch.helpers import scan
+from math import ceil, floor
+from multiprocessing.pool import ThreadPool
+from threading import Lock, get_ident
+
+
+_log = logging.getLogger(__name__)
```

Comment:
Since this is a management command, it should be "script" instead of the module name. Also the convention is to use `logger` instead of `_log`.


```suggestion
logger = logging.getLogger("script")
```

Reply:
I still don't think this is the best way to do this. 
By hard-coding this to a single logger named "script", there is no way to adjust log level of different modules executed _during_ the script run via the logging config in `settings.py`. 

The way this should be done is to have e.g. this (and other) management command modules all getting associated with the `"script"` handler by way of providing their module name as a logger in the `loggers` section of the logging config dict. If that becomes too verbose, that dict can be modified to pattern match (e.g. all modules that have `*.management.*` in their module namespace) to associate them with the `script` handler.